src/effects/native/balanceeffect.cpp

```diff
@@ -1,5 +1,9 @@
 #include "balanceeffect.h"
 
+namespace {
```

Comment:
What do you mean by "low pass mono bypass"? A low pass filter with a button parameter to bypass it? Could you explain what the use case is for combining that with this Balance effect? I have thought about hacking a low pass filter into the Bitcrusher effect after the bit crushing, but this feels inappropriate because we have a separate filter effect. Instead I have been putting a filter or Moog filter effect after the Bitcrusher effect in an EffectUnit. [Composable effect chains](https://bugs.launchpad.net/mixxx/+bug/1656189) would be a more flexible solution than hardcoded chains within individual effects. However, that will require a lot of work that I don't want to get into right now.